apps/dashboard/src/app/layout.tsx

```diff
@@ -1,22 +1,19 @@
+"use client"
 import type { PropsWithChildren } from "react"
 import { AuthProvider } from "./AuthProvider"
-import { unstable_getServerSession } from "next-auth"
-import { authOptions } from "../pages/api/auth/[...nextauth]"
 import { QueryProvider } from "./QueryProvider"
 import { MantineColorSchemeProvider, MantineProvider } from "./MantineProvider"
 import { ApplicationShell } from "./ApplicationShell"
 import "./root.css"
 
-export default async function RootLayout({ children }: PropsWithChildren) {
-  const session = await unstable_getServerSession(authOptions)
-
+export default function RootLayout({ children }: PropsWithChildren) {
   return (
     <html lang="en">
       <head />
       <body>
         <MantineColorSchemeProvider>
           <MantineProvider>
-            <AuthProvider session={session}>
+            <AuthProvider session={null}>
```

Comment:
What's going on here?

Reply:
Session is initially null until it's set through nextauth